### PR TITLE
Btc support dev

### PR DESCRIPTION
**Pull request by ghost**
Support BTC in SDK

### CODE REVIEW

File: src/api/ccUtil.js [hunk 1]
```diff
@@ -758,6 +775,244 @@ const ccUtil = {
...
+      return sum
+  },
+
+  /**
+   */
+  async getBtcUtxo(minconf, maxconf, addresses) {
+      let utxos = await this._getBtcUtxo(minconf, maxconf, addresses);
+      let utxos2 = utxos.map(function (item, index) {
+          let av = item.value ? item.value : item.amount;
+          item.value = Number(web3.toBigNumber(av).mul(100000000));
```
Comment (lansehuiyi6) [inline src/api/ccUtil.js:918]: the value use 'Number()', should it be change to toBigNumber(), in case of overflow

File: src/conf/config.js [hunk 2]
```diff
@@ -63,11 +85,12 @@ if (network === 'testnet') {
...
...ype":"bytes32"},{"indexed":false,"name":"x","type":"bytes32"}],"name":"InboundRefundLogger","type":"event"},{"anonymous":false,"inputs":[{"indexed":true,"name":"token","type":"address"},{"indexed":true,"name":"user","type":"address"},{"indexed":true,"name":"xHash","type":"bytes32"}],"name":"InboundRevokeLogger","type":"event"},{"anonymous":false,"inputs":[{"indexed":false,"name":"token","type":"address"},{"indexed":true,"name":"storeman","type":"address"},{"indexed":true,"name":"user","type":"address"},{"indexed":true,"name":"xHash","type":"bytes32"},{"indexed":false,"name":"value","type":"uint256"}],"name":"OutboundLockLogger","type":"event"},{"anonymous":false,"inputs":[{"indexed":false,"name":"token","type":"address"},{"indexed":true,"name":"user","type":"address"},{"indexed":true,"name":"storeman","type":"address"},{"indexed":true,"name":"xHash","type":"bytes32"},{"indexed":false,"name":"x","type":"bytes32"}],"name":"OutboundRefundLogger","type":"event"},{"anonymous":false,"inputs":[{"indexed":true,"name":"token","type":"address"},{"indexed":true,"name":"storeman","type":"address"},{"indexed":true,"name":"xHash","type":"bytes32"}],"name":"OutboundRevokeLogger","type":"event"}];
```
Comment (lansehuiyi6) [inline src/conf/config.js]: should delete the useless code

File: src/conf/config.js [hunk 3]
```diff
@@ -131,11 +161,12 @@ if (network === 'testnet') {
...
```
Comment (lansehuiyi6) [inline src/conf/config.js]: should delete the useless code

File: src/api/ccUtil.js (hunk 1 above)
Comment (ghost) [reply src/api/ccUtil.js:918]: In nodejs, the maximum safe integer is 2<sup>53</sup>-1, while the total bitcoin supply is around 21 million, which is about 2<sup>50.899</sup>; and the change may affect current use of it, so at this time, we may not change it

File: src/conf/config.js (hunk 3 above)
Comment (ghost) [reply src/conf/config.js]: Done

**Pull request merged**